test/runtests.jl

```diff
@@ -33,6 +33,16 @@ if GROUP == "All" || GROUP == "AdvancedHMC"
     end
 end
 
+if GROUP == "All" || GROUP == "Experimental"
```

Comment:
Let's create a new file `test/experimental/runtests.jl` and transfer these test changes there. The idea is that on the `hmc-research` branch, we only change files under `src/experimental` and `tests/experimental`. Everything else should remain the same with the `master` branch.